src/scripts/modules/media/body/content-common.less

```diff
@@ -130,6 +130,10 @@ figure[data-orient="vertical"] {
   font-weight: bold;
 }
 
+.no-emphasis {
```

Comment:
is this intended to be a change that applies to all of webview? Or should it be moved to content-baked.less?

Reply:
I put it in `content-common.less` because I figured that anytime there is a `.no-emphasis` it shouldn't be bolded, BUT I keep forgetting about the testing implications. I should probally apply this to only baked books correct, no matter how small the change? @helenemccarron 
cc @Stackblocks

Reply:
I seem to remember going back and forth on this with Alina, I may be wrong though. cc @openstaxalina